scripts/new_upstream_snapshot.py

```diff
@@ -349,6 +349,9 @@ def refresh_patches(commitish) -> bool:
             f"patches: \n" + "\n".join(patches)
         )
         sh(f"git commit --no-verify -m '{commit_msg}' {' '.join(patches)}")
+        add_msg_to_changelog(
+            "  * refresh patches:\n    - " + "\n    - ".join(patches)
```

Comment:
This makes for a lot of duplicated full file path names.
```
 * refresh patches
    - debian/patches/X.patch
    - debian/patches/Y.patch
    - debian/patches/Z.patch
```
How about we reduce this to 
```
 * refresh debian/patches:
    - X.patch
    - Y.patch
    - Z.patch
```

```diff
--- a/scripts/new_upstream_snapshot.py
+++ b/scripts/new_upstream_snapshot.py
@@ -14,6 +14,7 @@ The commitish defaults to 'main'
 """
 
 import argparse
+import os
 import re
 import subprocess
 import sys
@@ -350,7 +351,8 @@ def refresh_patches(commitish) -> bool:
         )
         sh(f"git commit --no-verify -m '{commit_msg}' {' '.join(patches)}")
         add_msg_to_changelog(
-            "  * refresh patches:\n    - " + "\n    - ".join(patches)
+            "  * refresh debian/patches:\n    - " +
+            "\n    - ".join([os.path.basename(p) for p in patches])
         )
         return True
     else:
```

Reply:
Gahhh, sorry. I had actually already changed this locally and rebased it, but never pushed it. I did a force push for it.